SPECS/xapi.spec

```diff
@@ -93,6 +94,9 @@ Requires: gmp
 Requires: zstd
 Requires: xxhash-libs
 Requires: jemalloc
+Requires: tdb-tools >= 1.3.18
+Requires: samba-winbind >= 4.10.16
+Requires: upgrade-pbis-to-winbind
```

Comment:
We should consider *not* adding those and wait for winbind to really replace PBIS in a potential CH 8.2.2, CH 8.3 or other.